docs/ref/settings.txt

```diff
@@ -1873,10 +1873,9 @@ for :doc:`managing stored files </topics/files>`. It must end in a slash if set
 to a non-empty value. You will need to :ref:`configure these files to be served
 <serving-uploaded-files-in-development>` in both development and production.
 
-In order to use ``{{ MEDIA_URL }}`` in your templates, you must have
+If you want to directly use ``{{ MEDIA_URL }}`` in your templates, add
```

Comment:
why "directly"? (e.g. is there some way to use it "indirectly"?) I presume you're thinking of `{{ file.url }}` but I'm not sure that's important here.